discord/guild.py

```diff
@@ -314,6 +314,7 @@ def _from_data(self, guild):
         self.discovery_splash = guild.get('discovery_splash')
         self._rules_channel_id = utils._get_as_snowflake(guild, 'rules_channel_id')
         self._public_updates_channel_id = utils._get_as_snowflake(guild, 'public_updates_channel_id')
+        self.nsfw = guild.get('nsfw')
```

Comment:
Should probably default to False here rather than None

Reply:
Changed it.